### ISSUE TITLE
Add Earth Wallet

### ISSUE THREAD

**Issue by everhusk** [ethereum/ethereum-org-website#12529]
### Wallet name

Earth Wallet

### Wallet description

Earth Wallet is the self-custody Bitcoin & Ethereum wallet for keeping full control over their assets. Safely store, stake, and grow your Ethereum holdings anywhere on Earth with ease.

### Wallet logo

![logo](https://github.com/ethereum/ethereum-org-website/assets/4977840/02cf370e-f632-4553-8bf5-88bdfb1b99b8)

### Background color for brand logo

000000

### URL to the project

https://earthwallet.io

### When did the wallet go live to users?

July 2022

### Does the wallet have an active development team?

Yes github.com/earthwallet

### Is the wallet globally accessible?

Available everywhere on Earth

### Is the wallet available in multiple languages?

Yes, (AR, BN, EN, ES, FE, HI, MR, RU, SW, TA, TE, TH, UR, VI, and ZH)

### What social links are there for the project?

https://x.com/earthwallet
https://instagram.com/earth_wallet

### Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?

Yes it is available on iOS and Android.

### Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?

No.

### Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?

Yes it has an Open Source Chrome app here - https://github.com/earthwallet/wallet

### Is it a hardware wallet?

No

### Is the source code for the wallet fully open-source?

Yes github.com/earthwallet/wallet

### What license is the wallet software released under?

GNU General Public License v3.0

### Who holds the private keys?

The user holds all private keys, encrypted by device biometrics or a password for non biometric enabled devices.

### Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible

We have completed a full independent security audit of the code by Halborn security, the results are available here: [Earth_Wallet_Android_iOS_Mobile_App_Pentest_Report_Halborn_Final.pdf](https://github.com/ethereum/ethereum-org-website/files/13497580/Earth_Wallet_Android_iOS_Mobile_App_Pentest_Report_Halborn_Final.pdf)

### Has the wallet's smart contract code or security modules been audited?

Yes see above.

### Does the wallet have an internal security team?

Yes, our team has over a decade of experience building and securing over $1B in digital assets (TradFi, CEXs, and DeFi).

### Any other security testing that should be noted?

We are always reviewing the latest security best practices, simplifying our code, and looking to advance our security defence.

### Scam protection?

Earth Wallet always prompts the user before requesting any signature or dapp connection.

### Does the wallet support connecting to a hardware wallet?

No

### Does the wallet support WalletConnect?

Yes

### Does the wallet support importing Ethereum RPC endpoints?

Yes

### Does the wallet support viewing and interacting with NFTs?

Yes

### Does the wallet support connecting to Ethereum applications?

Yes, users can connect using wallet connect v2 on all platforms (iOS, Android, and Chrome) via QR code or browser.

### Does the wallet support staking directly?

Yes.

### Does the wallet support swaps directly?

It is planned in a future release.

### Does the wallet support multi-chain networks?

Yes, Earth Wallet support Bitcoin, and Social Network, a new Layer 2 BTC/ETH Rollup.

### Does the wallet support Ethereum layer 2 networks?

Yes Social Network

### Does the wallet allow the user to customize gas fees?

Yes. Users can select from 3 options or customize.

### Does the wallet support sending transactions to ENS addresses?

Yes, Earth Wallet supports ENS and unstoppable domain address resolution and reverse resolution.

### Does the wallet support importing or automatically querying and displaying ERC-20 tokens?

Yes, it supports automatically displaying ERC20 tokens.

### Does the wallet support EIP-1559 (type 2) transactions?

No.

### Does the wallet have fiat on-ramps through credit/debit cards, wire transfers, or bank transfers (ACH)?

No.

### Does the wallet support withdrawals to fiat?

No.

### Is the wallet a multi-signature wallet?

It is a planned release later this year.

### Does the wallet support social recovery?

It is a planned release later this year.

### Who can the ethereum.org team can contact regarding the wallet in future?

t.me/everhusk
x.com/everhusk

### Does the wallet have a dedicated support team?

t.me/earthwallet

### What educational resources/documentation do you provide to users?

docs.earthwallet.io

### Does the wallet have any integrated tools not mentioned above?

We will enable seamless interoperability between Bitcoin and Ethereum.

### Would you like to work on this issue?

- [ ] Yes
- [X] No

**Comment by wackerow** [comment]
Thanks @everhusk for the request! Just took a quick glimpse through and looks like everything is here... Assigning @corwintines to formally review, and if everything looks good we can get a PR up for this. Bear with us on timing though as product listings are a lower priority in general for this site where a lot is going on in parallel. 

---

Also, 

> Does the wallet support EIP-1559 (type 2) transactions?
> No.

would provide cheaper more predictable fees for users; would recommend =)

**Comment by konopkja** [comment]
Closing the issue for now as 1559 is not supported and we also have updated our wallet form to require additional info.
